conans/client/cmd/new.py

```diff
@@ -281,14 +310,30 @@ def cmd_new(ref, header=False, pure_c=False, test=False, exports_sources=False,
         files = {"conanfile.py": conanfile_bare.format(name=name, version=version,
                                                        package_name=package_name)}
     elif template:
-        if not os.path.isabs(template):
-            template = os.path.join(cache.cache_folder, "templates", template)
-        if not os.path.isfile(template):
-            raise ConanException("Template doesn't exist: %s" % template)
-        conanfile_template = load(template)
-        t = Template(conanfile_template)
-        replaced = t.render(name=name, version=version, package_name=package_name)
-        files = {"conanfile.py": replaced}
+        is_file_template = os.path.basename(template).endswith('.py')
+        if is_file_template:
+            if not os.path.isabs(template):
+                # FIXME: Conan 2.0. The old path should be removed
+                old_path = os.path.join(cache.cache_folder, "templates", template)
+                new_path = os.path.join(cache.cache_folder, "templates", "cmd_new", template)
```

Comment:
Hi @TheQwertiest 

Please, lets use the path ``templates/command/new``. 

The reason is that in Conan 2.0 we are going to modularize the commands, making it an extensible framework to plug user commands. So it will be cleaner to have all of those things inside a ``templates/command/`` subfolder, then followed by the name of the command.

Many thanks!

Reply:
Sure

Reply:
Done.

PS: Sorry for nagging, but I would really appreciate if you answer a question about Python 2.7 in meson helper PR =(